content/en/docs/reference/.where-are-the-docs

```diff
@@ -0,0 +1,4 @@
+# AAAAA?! Where be the docs?
+
+API Docs are autogenerated from the Golang source.
+Go edit the source comments [here](https://github.com/jetstack/cert-manager/tree/master/pkg/internal/apis) to update API docs.
```

Comment:
Can we make this the versioned API docs as we document those and not the internal one, which is only used for conversion logic.

Reply:
@TheEnbyperor Here is the link you can use:
```suggestion
Go edit the source comments [here](https://github.com/jetstack/cert-manager/tree/master/pkg/apis) to update API docs.
```